src/NzbDrone.Core/Parser/ParseUtil.cs

```diff
@@ -113,8 +113,16 @@ public static string GetArgumentFromQueryString(string url, string argument)
 
         public static long GetBytes(string str)
         {
+            var valStr = new string(str.Where(c => char.IsDigit(c) || c == '.' || c == ',').ToArray());
```

Comment:
We already handle this functionality with `NormalizeNumber`

Reply:
I reverted my change since i just saw its already implemted 😄